tox.ini

```diff
@@ -30,8 +30,6 @@ extras =
     development
     pandas: pandas
     sso: secure-local-storage
-deps =
-    pip >= 19.3.1
```

Comment:
Do we not need to depend on pip any more?

Reply:
No, `setuptools`, `pip` and `wheel` are all installed by default